ubuntu_mint.sh

```diff
@@ -74,11 +78,24 @@ function nuc_data_make {
 }
 
 function test_pyne {
+    
+    # only test for python version if using the most recent dev branch
+    if [ $1 == 'dev' ] ; then
```

Comment:
Is this condition only relevant at this moment in time?  i.e. when `stable` supports both versions, will this need to go away - or be extended to all cases?

Reply:
Correct, it is only relevant right now. But I am not sure how long this will be the case. I think it is best to have the install scripts working with the current situation and then update them when a new release comes out. It is unclear to me how long until the next release, so I think having this here for now is a good idea. It is an easy change when then new release comes out to get rid of this requirement.